mautic/mautic
# Stackedbars chart

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
this PR adds dynamic width to barcharts

#### Steps to test this PR:
1. go to email for segments with a bunch of segments bars should pretty much fall into place
2. check the lifecycle chart
